curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/check-for-groups-of-characters.english.md

```diff
@@ -0,0 +1,65 @@
+---
+id: 587d7dbb367417b2b2512be6
+title: Check for Groups of Characters
+challengeType: 1
+---
+
+## Description
+<section id='description'>
+Sometimes we want to check for groups of characters using a Regular Expression, to achieve this task we use the round brackets <code>()</code>.
+If you want to find either <code>"Hello"</code> or <code>"Bye"</code> in a string, you could use the following Regular Expression: <code>/(Hello|Bye)/</code>
```

Comment:
Both here, in the example and task, the grouping isn't needed. It has to be against an encased substring for it to matter, like the example I gave in the bug report.

> testStr = "freeCodeCamp"; testRegex = /Code|Programming/;testStr.match(testRegex);
> ["Code", index: 4, input: "freeCodeCamp", groups: undefined]

Reply:
All these are outdated now. Check the new files